### PR TITLE
chore(docs): update JetBrains gateway setup

### PR DESCRIPTION
**Pull request by EdwardAngert**
- updates the JetBrains Gateway set up steps
- adjusts the relative image urls to the new directory structure

### CODE REVIEW

File: docs/user-guides/workspace-access/jetbrains.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ IDEs are supported for remote development:
 
 JetBrains Gateway is a compact desktop app that allows you to work remotely with
 a JetBrains IDE without even downloading one.
-[See JetBrains' website to learn about and Gateway.](https://www.jetbrains.com/remote-development/gateway/)
+[See JetBrains's website to learn about and Gateway.](https://www.jetbrains.com/remote-development/gateway/)
```
Comment (matifali) [inline docs/user-guides/workspace-access/jetbrains.md:20]: As the last letter is an `s` we don't need an extra apostrophe s

**Review by matifali — APPROVED** [review]
Thank you. There is only 1 observation regarding an apostrophe s.

File: docs/user-guides/workspace-access/jetbrains.md (hunk 1 above)
Comment (EdwardAngert) [inline docs/user-guides/workspace-access/jetbrains.md:20]:
```suggestion
Visit the [JetBrains website to learn about and Gateway.](https://www.jetbrains.com/remote-development/gateway/)
```

Comment (EdwardAngert) [reply docs/user-guides/workspace-access/jetbrains.md]: [Google Developer Documentation Style Guide](https://developers.google.com/style/possessives) suggests avoiding it altogether, so I adjusted this line

**Pull request merged**